hnn_core/parallel.py

```diff
@@ -7,20 +7,76 @@
 
 from neuron import h
 
-rank = 0
-nhosts = 1
-pc = h.ParallelContext(nhosts)
-pc.done()
-rank = int(pc.id())
-cvode = h.CVode()
+pc = None
+last_network = None
 
 
-def create_parallel_context(n_jobs=1):
-    """Create parallel context."""
-    rank = int(pc.id())     # rank or node number (0 will be the master)
+def shutdown():
+    """
+    Tell nrniv processes to terminate. Calling script will also terminate
+    """
```

Comment:
nitpick but can we use the docstring conventions [here](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings). For one line docstrings, the triple quotes should all fit in one line

Reply:
It's too long for one line. I'll start the text on the first line, okay?

Reply:
Err... was too many characters before. Fits on one line now